### PR TITLE
Create am730.com.hk.xml

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

File: src/chrome/content/rules/am730.com.hk.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		Couldn't resolve host name:
+			 - am730.com.hk
+			 - mailserv.am730.com.hk
+			 - www1.am730.com.hk
+
+		Timeout was reached:
+			 - epaper.am730.com.hk
+
+		Secure connection redirects to plaintext:
+			 - archive.am730.com.hk
```
Comment (J0WI) [inline src/chrome/content/rules/am730.com.hk.xml]: I couldn't reproduce this, but I have mixed content issues

Comment (J0WI) [inline src/chrome/content/rules/am730.com.hk.xml]: Please also add a comment for `flipbook.am730.com.hk` (serves different content)

File: src/chrome/content/rules/am730.com.hk.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+		Mixed content blocking (MCB) tiggered:
+			 - apps.am730.com.hk
+			 - m.am730.com.hk
+-->
+<ruleset name="am730.com.hk">
+	<target host="www.am730.com.hk" />
+	<target host="apps2.am730.com.hk" />
+	<target host="charityrun.am730.com.hk" />
+	<target host="flipbook.am730.com.hk" />
```
Comment (J0WI) [inline src/chrome/content/rules/am730.com.hk.xml]: This host serves different content for me

File: src/chrome/content/rules/am730.com.hk.xml (hunk 1 above)
Comment (cschanaj) [inline src/chrome/content/rules/am730.com.hk.xml]: I didn't notice that `http(s)://archive.am730.com.hk/` always redirect to `http://archive.am730.com.hk/home` while the host does have a working HTTPS connection. You are correct about the MCB issues.

File: src/chrome/content/rules/am730.com.hk.xml (hunk 2 above)
Comment (cschanaj) [inline src/chrome/content/rules/am730.com.hk.xml]: You're right. Fixed. Thanks.

**Pull request merged**